server/matchmaker/search.py

```diff
@@ -16,6 +17,28 @@
 OnMatchedCallback = Callable[["Search", "Search"], Any]
 
 
+class Game:
```

Comment:
Not a fan of this name because we already have a class called `Game` in the `games` module.